data/categories.txt

```diff
@@ -8845,7 +8849,7 @@ el:^Κατάστημα ερωτικών ειδών
 es:^Sex Shop
 fi:^Erotiikkaliike
 fr:^Boutique érotique
-hu:^Erotikus bolt
+hu:^Erotikus bolt|szexbolt
```

Comment:
Isn't this attachment of "bolt" supposed to be written without extra spaces?

Reply:
​No, it is good so.